src/graphicalClient/soccerview.cpp

```diff
@@ -99,8 +107,7 @@ void GLSoccerView::mousePressEvent(QMouseEvent* event)
 
 void GLSoccerView::mouseReleaseEvent(QMouseEvent* event)
 {
-  bool shiftKey = event->modifiers().testFlag(Qt::ShiftModifier);
-  bool ctrlKey = event->modifiers().testFlag(Qt::ControlModifier);
+  (void)event;
```

Comment:
Does removing this result in an unused variable warning? If so, what compiler and version?